components/components/regions/header/header.twig

```diff
@@ -0,0 +1,29 @@
+{#
+/**
+ * @file
+ * This file creates the header for our styleguide.
+ *
+ * The header is made up of a number of smaller components - blocks
+ */
+#}
+
+<header class="header" role="banner">
+  <div class="region region-header container">
+    {% block content %}
+      {#
+        In our Drupal template, we need to replace these two include files
+        with {{ content }}, so we print whatever blocks are placed in that
+        region.
+
+        Also, I'm not happy with having to duplicate the content from the
```

Comment:
Do you think we could explain this with a [@todo ](https://www.drupal.org/docs/develop/coding-standards/api-documentation-and-comment-standards#todo)note instead a first person comment?

Reply:
Done. But it might not be needed if we are moving away from the styleguide and towards a normal Drupal theme.